web3j/src/main/java/com/vmware/ethereum/config/AppConfig.java

```diff
@@ -103,6 +116,34 @@ public Credentials credentials() throws IOException, CipherException {
         credentials.getWalletPassword(), credentials.getWalletFile());
   }
 
+  @Bean
+  @ConditionalOnExpression("${config.isQueuedPolling():true}")
```

Comment:
This looks more simpler. Try if you want.
@ConditionalOnProperty(value="config.isQueuedPolling", havingValue = "true")

Reply:
yeah will update this.

Reply:
Reverted to @ConditionalOnExpression("${config.isQueuedPolling():true}"), as @ConditionalOnProperty(value="config.queued-polling", havingValue = "true") was not working properly.

Reply:
done

Reply:
I don't see the code